Cloud change log update (#817)
SkylineCommunications/dataminer-docs: release-notes/Cloud/DCP_change_log.md

The DataMiner Cloud Platform gets updated continuously. This change log can help you trace when specific features and changes have become available.

> #### 9 December 2022 - Enhancement - FieldControl 2.8.2 - Newtonsoft dependencies updated to 13.0.2 [ID_35140]

Reviewer: TODO

- The column order has been adjusted.
- Automatic loading of audit records has been improved to prevent possible issues with different screen sizes.

> #### 15 September 2022 – Fix – CloudGateway 2.9.5 – Problem in CloudGateway process if MaintenanceSettings.xml contained invalid URLs [ID_34341]

Reviewer: TODO

> #### 15 September 2022 – Fix – CloudGateway 2.9.5 – Problem in CloudGateway process if MaintenanceSettings.xml contained invalid URLs [ID_34341]

Reviewer: ```suggestion
#### 15 September 2022 – Fix – CloudGateway 2.9.5 – Problem in CloudGateway process if MaintenanceSettings.xml contained invalid an invalid HTTPS endpoint [ID_34341]
```

> If URLs in the file *MaintenanceSettings.xml* were not configured correctly, a problem could occur in the CloudGateway process. This happened specifically when DataMiner upgrades caused the HTTP URL to end with a new line.

Reviewer: ```suggestion
If the HTTPS endpoint in the file *MaintenanceSettings.xml* was not configured correctly, a problem could occur in the CloudGateway process. This happened specifically when DataMiner upgrades caused the HTTPS URL to end with an escaped new line.
```

> With CloudGateway 2.9.5 (included in Cloud Pack version 2.8.2), all URLs from configuration files will be trimmed to prevent this, so that CloudGateway will no longer get this problem at runtime. However, note that CloudGateway may still fail to start up if a URL in a configuration file is invalid.

Reviewer: ```suggestion
With CloudGateway 2.9.5 (included in Cloud Pack version 2.8.2), all endpoints from configuration files will be trimmed to prevent this, so that CloudGateway will no longer get this problem at runtime. However, note that CloudGateway may still fail to start up if a URL in a configuration file is invalid.
```

> #### 15 September 2022 – Fix – CoreGateway 2.11.5 – Incorrect information event when connector was deployed [ID_34325]

Reviewer: when connector was deployed .... from/via DCP

The Cloud Gateway now comes with a new connection tester tool, *ConnectionTester.exe*. This tool can be used to validate the network setup and check if all features are available. It checks whether the network complies with the requirements for the cloud platform.

You can find the new tool in the folder `Program files\Skyline Communications\Dataminer CloudGateway\` on a DMA that has the Cloud Gateway installed. Run the executable as administrator. The connection tester will connect to port 443 to check whether requirements are met, and it will show the results in a console window.

This feature is included in Cloud Pack version 2.8.2.

#### 1 September 2022 – Fix – CoreGateway 2.11.4 / CloudGateway 2.9.4 – Login screen shown when not necessary while viewing share or using remote access [ID_34275]

When the DataMiner Cloud Pack was installed in a cluster with two or more DMAs, it could occur that users trying to view a shared dashboard or remotely access a DMS could be shown the login screen when this was not supposed to happen.

With CoreGateway 2.11.4 and CloudGateway 2.9.4 (included in Cloud Pack version 2.8.2), this issue is resolved.

> The CoreGateway DxM must now be installed on the same DMA as CloudGateway to ensure that sharing and remote access will work correctly. For DMZ setups, the DMA that the cloud gateway points to will need to have the CoreGateway DxM installed.

Reviewer: ```suggestion
The CoreGateway DxM must now be installed on the same DMA as CloudGateway to ensure that sharing and remote access will work correctly. For DMZ setups, the DMA that the CloudGateway  points to will need to have the CoreGateway DxM installed.
```

#### 15 September 2022 – Fix – CloudGateway 2.9.5 – Problem in CloudGateway process if MaintenanceSettings.xml contained invalid URLs [ID_34341]

If URLs in the file *MaintenanceSettings.xml* were not configured correctly, a problem could occur in the CloudGateway process. This happened specifically when DataMiner upgrades caused the HTTP URL to end with a new line.

With CloudGateway 2.9.5 (included in Cloud Pack version 2.8.2), all URLs from configuration files will be trimmed to prevent this, so that CloudGateway will no longer get this problem at runtime. However, note that CloudGateway may still fail to start up if a URL in a configuration file is invalid.

#### 15 September 2022 – Fix – CoreGateway 2.11.5 – Incorrect information event when connector was deployed [ID_34325]

> When a connector was deployed via DCP, the corresponding information event incorrectly mentioned that this was done with the SLNetClientTest tool.

Reviewer: maybe also in the title?
